gemrb/core/GameScript/Actions.cpp

```diff
@@ -6864,8 +6864,10 @@ void GameScript::UseItemPoint(Scriptable* Sender, Action* parameters)
 		return;
 	}
 
+	Point targetPos = parameters->pointParameter;
+
 	Sender->ReleaseCurrentAction();
-	act->UseItemPoint(Slot, header, parameters->pointParameter, flags);
+	act->UseItemPoint(Slot, header, targetPos, flags);
```

Comment:
It is better to document such changes explicitly in the code so them wouldn't be refactored back.

Reply:
It is pretty obvious if you're working on actions code.

Reply:
And yet, the bug appeared, for all the obvious

Reply:
I agree with you @Mingun, but I already have a WIP patch to use `shared_ptr` for `Actions` (see #1913) which would solve the possibility of this sort of bug appearing again which is probably better than a comment.